app/helpers/account.ts

```diff
@@ -0,0 +1,28 @@
+import { Server } from "stellar-sdk";
+const server: Server = new Server("https://horizon-testnet.stellar.org");
+
+const loadAccount = async (publicKey: string) => {
+  try {
+    const account = await server.loadAccount(publicKey);
+    return account;
```

Comment:
```suggestion
    return await server.loadAccount(publicKey);
```

Reply:
I have removed the intermediate constant and now the function returns the method call directly.